iroha_p2p/src/chacha.rs

```diff
@@ -0,0 +1,66 @@
+use std::fmt;
```

Comment:
Could you do this in another pr and just abstract over encryption itself? You can make traits for generating keys, nonce and all that stuff

Reply:
Yep!